Index: src/test/resources/features/UserStory5_YAVUZ.feature
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>#As a librarian, I should be able to see book records on user page\n#\n#\n#AC:  1. default 10 records\n#2. Show records for <count> options\n#Examples:\n#| count |\n#| 5        |\n#| 10      |\n#| 15      |\n#| 50      |\n#| 100    |\n\n@login\nFeature: A librarian should be able to see book records on user page\n  @login @wip\n  Scenario:\n    Given the user is on the login page\n    When  the user enters the \"Librarian\" information\n    Then  the Library page should be displayed
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/resources/features/UserStory5_YAVUZ.feature	(revision 5ac4f63bf8aa11eb54e8dceeb337127fce7584ab)
+++ src/test/resources/features/UserStory5_YAVUZ.feature	(date 1604936608438)
@@ -17,4 +17,6 @@
   Scenario:
     Given the user is on the login page
     When  the user enters the "Librarian" information
-    Then  the Library page should be displayed
\ No newline at end of file
+    Then  the Library page should be displayed
+    When  the user clicks on "Users" module on the Library page
+    Then  the Users Management page should be displayed
\ No newline at end of file
