Index: src/test/resources/features/UserStory5_YAVUZ.feature
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>#As a librarian, I should be able to see book records on user page\n#\n#\n#AC:  1. default 10 records\n#2. Show records for <count> options\n#Examples:\n#| count |\n#| 5        |\n#| 10      |\n#| 15      |\n#| 50      |\n#| 100    |
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/resources/features/UserStory5_YAVUZ.feature	(revision 473f6d45b69ebd96157a8d8b70ad9e5a4ab08e16)
+++ src/test/resources/features/UserStory5_YAVUZ.feature	(date 1604936722054)
@@ -9,4 +9,14 @@
 #| 10      |
 #| 15      |
 #| 50      |
-#| 100    |
\ No newline at end of file
+#| 100    |
+
+@login
+Feature: A librarian should be able to see book records on user page
+  @login @wip
+  Scenario:
+    Given the user is on the login page
+    When  the user enters the "Librarian" information
+    Then  the Library page should be displayed
+    When  the user clicks on "Users" module on the Library page
+    Then  the Users Management page should be displayed
