Index: src/test/java/com/Library/Pages/BooksPage.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.Library.Pages;\n\nimport com.Library.Utilities.Driver;\nimport org.openqa.selenium.support.PageFactory;\n\npublic class BooksPage extends BasePage{\n\n    public BooksPage() {\n        PageFactory.initElements(Driver.get(), this);\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/java/com/Library/Pages/BooksPage.java	(revision d3511a3d799dc8cd12139161599f8d355ff4412f)
+++ src/test/java/com/Library/Pages/BooksPage.java	(date 1604936866598)
@@ -3,7 +3,7 @@
 import com.Library.Utilities.Driver;
 import org.openqa.selenium.support.PageFactory;
 
-public class BooksPage extends BasePage{
+public class BooksPage extends BasePage {
 
     public BooksPage() {
         PageFactory.initElements(Driver.get(), this);
Index: src/test/resources/features/UserStory5_YAVUZ.feature
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>#As a librarian, I should be able to see book records on user page\n#\n#\n#AC:  1. default 10 records\n#2. Show records for <count> options\n#Examples:\n#| count |\n#| 5        |\n#| 10      |\n#| 15      |\n#| 50      |\n#| 100    |
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/test/resources/features/UserStory5_YAVUZ.feature	(revision d3511a3d799dc8cd12139161599f8d355ff4412f)
+++ src/test/resources/features/UserStory5_YAVUZ.feature	(date 1604936724679)
@@ -9,4 +9,14 @@
 #| 10      |
 #| 15      |
 #| 50      |
-#| 100    |
\ No newline at end of file
+#| 100    |
+
+@login
+Feature: A librarian should be able to see book records on user page
+  @login @wip
+  Scenario:
+    Given the user is on the login page
+    When  the user enters the "Librarian" information
+    Then  the Library page should be displayed
+    When  the user clicks on "Users" module on the Library page
+    Then  the Users Management page should be displayed
